app/main.py

```diff
@@ -2,4 +2,55 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.coords = [0, 0] if coords is None else coords
```

Comment:
You could omit `is None` check here
```suggestion
        self.coords = coords if coords else [0, 0]
```